src/wp-includes/blocks/latest-comments.php

```diff
@@ -42,7 +42,7 @@ function wp_latest_comments_draft_or_post_title( $post = 0 ) {
  *
  * @return string Returns the post content with latest comments added.
  */
-function render_block_core_latest_comments( $attributes = array() ) {
+function render_block_core_latest_comments( $attributes ) {
```

Comment:
Why these changes needed?

Reply:
@mukeshpanchal27 Please check ticket description, I've added clarification in core ticket.